### PR TITLE
Describe me query

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3440 Describe me query in Users section](https://app.clickup.com/t/2549495/SALEOR-3440)

File: docs/developer/users.mdx [hunk 1]
```diff
@@ -388,6 +388,45 @@ mutation {
...
+        "country": {
+          "country": "United States of America"
+        }
+      }
+    }
+  }
+}
+```
+
+If request is missing `authorization` header or, server will response with `null`.
```
Comment (korycins) [inline docs/developer/users.mdx:428]: `If request is missing authorization header or,`  Did you miss here something?

Comment (krzysztofwolski) [inline docs/developer/users.mdx:428]:
```suggestion
If the request is missing the `authorization` header or the token is invalid, the server will respond with `null`.
```

**Review by korycins — APPROVED** [review]

File: docs/developer/users.mdx [hunk 2]
```diff
@@ -388,6 +388,45 @@ mutation {
 }
 ```
 
+## Accessing authorizated user info
```
Comment (patrys) [inline docs/developer/users.mdx:391]:
```suggestion
## Accessing information about the authenticated user
```

"Authentication" is about your identity, "authorization" is about your privileges.

File: docs/developer/users.mdx [hunk 3]
```diff
@@ -388,6 +388,45 @@ mutation {
...
+
+[`me`](developer/api-reference/queries/me.mdx) allow you to get information about currently logged in user.
```
Comment (karolkielecki) [inline docs/developer/users.mdx:393]:
```suggestion
The [`me`](developer/api-reference/queries/me.mdx) query allows you to get information about currently logged in user. 
```

**Review by karolkielecki — CHANGES_REQUESTED** [review]

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**